<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget_1" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>630</height>
     </rect>
    </property>
    <widget class="QPushButton" name="btnAddVertex">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>140</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Agregar Estado</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkIsAcceptation">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Estado de aceptación</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkIsStart">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>160</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Nodo de partida</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineVertex">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QListView" name="listViewVertex">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>430</width>
       <height>411</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineRead">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>40</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineExtract">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>80</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="linePutInHeap">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>120</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonAddRules">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>170</y>
       <width>140</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Agregar reglas</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>500</width>
      <height>630</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="lineWords">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonValidate">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>100</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Validar</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMessage">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>300</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>NO ACEPTACIÓN</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
